modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -66,7 +80,30 @@ do
 done
 ----
 
-. After the nodes in your cluster have rebooted, confirm that the migration succeeded:
+. After the nodes in your cluster have rebooted, start all of the machine configuration pools:
++
+[source,terminal]
+----
+$ oc patch MachineConfigPool master --type='merge' --patch \
+  '{ "spec": { "paused": false } }'
+----
++
+[source,terminal]
+----
+$ oc patch MachineConfigPool worker --type='merge' --patch \
+  '{ "spec": { "paused": false } }'
+----
+
+. To confirm the status of the new machine configuration on a host, enter the following command. Replace `<node>` with the name of the node.
++
+[source,terminal]
+----
+$ oc describe <node> | egrep "hostname|machineconfig"
```

Comment:
Just use   oc describe node | egrep "hostname|machineconfig",  it will list all the nodes status.